### DISCUSSION TITLE
Updating 4.6.0 to 5.x

### DISCUSSION THREAD

**Discussion by joerg376** [Athou/commafeed#1538] [Q&A] [answered]
Hi there,

running 4.6.0 in a docker container on my Synology NAS (DS916+), trying to upgrade to version 5.x
Edited docker compose (username/password for db), stack get's deployed without errors but I can't access the web interface anymore, getting a connection reset message.

Checked the logs from both docker containers in my stack but don't see any errors. Anybody with identical setup who got this working? Could really need some advice on this... ;-)

Thanks and Cheers,
J

**Comment by Athou** [comment]
Could you paste your CommaFeed logs please? It's hard to help with so little information :)

**Comment by joerg376** [comment]
Hi and thx for your answer.
Sure, will post more information (sorry, missed that in my initial post).
Please find the logs as well my docker-compose attached.

I'm running all my docker containers behind Synology Reverse Proxy but even w/o I can't get version 5.x running.
[commafeed-db_logs.txt](https://github.com/user-attachments/files/16838765/commafeed-db_logs.txt)
[commafeed_logs.txt](https://github.com/user-attachments/files/16838768/commafeed_logs.txt)
[docker-compose-4.txt](https://github.com/user-attachments/files/16838778/docker-compose-4.txt)
[docker-compose-5.txt](https://github.com/user-attachments/files/16838779/docker-compose-5.txt)

Thanks for any hints.

Cheers,
J

**Reply by Athou** [reply]
I think you may have [this issue](https://github.com/Athou/commafeed/issues/1532).

I still need to release a new version to fix this, probably later today. In the mean time, could you please test with the `master-postgresql` docker image?

**Comment by joerg376** [comment]
Thanks for your lightning-fast replies. :-)
![grafik](https://github.com/user-attachments/assets/5478a486-5075-40ca-8ffd-b2f0f2b5ae43)
It's working now with master-postgresql - thanks!

Cheers,
J

**Reply by Athou** [reply, marked as answer]
5.1.0 has been released with the fix!

**Comment by joerg376** [comment]
Hi. Just edited my docker-compose to use "latest-postgresql". Working! :-)
Thanks again for your fast support, much appreciated.

Cheers,
J
